src/.nuget/Microsoft.NETCore.TestHost/Microsoft.NETCore.TestHost.builds

```diff
@@ -39,6 +39,10 @@
       <OSGroup>Linux</OSGroup>
       <Platform>amd64</Platform>
     </Project>
+    <Project Condition="'$(TargetsLinux)' == 'true' and '$(DistroRid)' == 'fedora.27-x64'" Include="fedora/24/Microsoft.NETCore.TestHost.pkgproj">
```

Comment:
In this file, the includes also seem to have distro version mismatch at multiple places.